ui/js/graphs.js

```diff
@@ -472,6 +479,13 @@ perf.controller('GraphsCtrl', [
                         return $scope.zoom;
                     }
                 })(),
+                tooltip: (function() {
+                    var retTooltip = ($scope.selectedDataPoint) ? "["
+                        + $scope.selectedDataPoint.projectName + "," + $scope.selectedDataPoint.signature
+                        + "," + $scope.selectedDataPoint.resultSetId + "," + $scope.selectedDataPoint.flotDataOffset
```

Comment:
Sorry, after thinking this over a bit more, I'm not sure if this is right. I don't think we should be persisting flotDataOffset -- instead we should be persisting the job id of the selectedDataPoint to the tooltip, then looking that up on the graph. Otherwise I think the selected data point could be wrong in the case that we retrigger the test on the same result set. This involves adding a bunch more code to start tracking job ids of performance data points, which I don't believe we're doing yet.